### ISSUE TITLE
mvec subtraction always returns zero

### ISSUE THREAD

**Issue by serin-delaunay** [Zuzu-Typ/PyGLM#96]
When I access rows from a matrix as vectors I get correct values, but can't correctly subtract them without an additional step. Example:

```
import glm
a = glm.fvec(0.1,0.2)
b = glm.fvec(0.5,0.4)

print(a-b)
# vec2(-0.4,-0.2)

m = glm.fmat2x2(a,b)
print(m[0])
# mvec2(0.1, 0.2)
print(m[1])
# mvec2(0.5, 0.4)

print(m[0] - m[1])
# vec2(0,0) # incorrect
```

I get the correct answer if I first convert these mvecs back to fvecs.

**Comment by serin-delaunay** [comment]
mvec addition seems to double the left operand; I suspect the operations are receiving a second copy of it in place of the right operand.

**Comment by Zuzu-Typ** [comment]
Hi there Serin,

the issue you mentioned is caused by a few copy-and-paste errors I made when last updating mvec's methods.
The test system is designed to find these errors, but there were no tests for mvec up until this point.

Thank you for letting me know about it.
It should be fixed by #99
